### PR TITLE
Google Play sub-commands

### PR DESCRIPTION
**Pull request by Doges**
- [x] App
- [x] Developer

### CODE REVIEW

**Comment by Doges** [comment]
i wanna die while doing this tbh

**Comment by Doges** [comment]
RFR

**Comment by Doges** [comment]
now its rfr

File: src/locales/en-US/commands.json [hunk 1]
```diff
@@ -1714,21 +1714,29 @@
...
+        "trackInfo": "APP information"
+      }
+    }
   },
+    "rolelist": {
+      "commandDescription": "Lists all the roles in the server.",
+      "roles": "Roles",
+      "ids": "IDs",
+      "loading": "Loading roles..."
+    },
```
Comment (almeidx) [inline src/locales/en-US/commands.json:1739]:
```suggestion
  "rolelist": {
    "commandDescription": "Lists all the roles in the server.",
    "roles": "Roles",
    "ids": "IDs",
    "loading": "Loading roles..."
  },
```

**Review by almeidx — DISMISSED** [review]

File: src/locales/en-US/commands.json [hunk 2]
```diff
@@ -1714,21 +1714,29 @@
     "guildHasNoEmoji": "This server has no available emojis for me."
   },
   "googleplay": {
-    "commandDescription": "Searches for an APP on Google Play Store",
-    "commandUsage": "<application name>",
+    "commandDescription": "Searches for an application, developer or [REDACTED] on Google Play Store",
```
Comment (almeidx) [inline src/locales/en-US/commands.json:1717]: Redacted?

Comment (Doges) [reply src/locales/en-US/commands.json:1717]: OH, I FORGOT TO REMOVE

**Review by almeidx — CHANGES_REQUESTED** [review]
The developer subcommand does not have commandDescription or commandUsage

**Review by almeidx — DISMISSED** [review]

File: src/commands/misc/googleplay/app.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    return `[${items.title} - ${items.developer}](${items.url})`
+  }
+
+  async handleResult ({ t, channel, author, language }, { title, screenshots, installs, released, contentRating, genre, url, priceText, price, icon, summary, appId, developer, developerId, currency }) {
+    channel.send(
+      new SwitchbladeEmbed(author)
+        .setColor(this.embedColor)
+        .setThumbnail(icon)
+        .setAuthor(title, icon, url)
+        .setDescription(summary)
```
Comment (jaxxibae) [inline src/commands/misc/googleplay/app.js:28]:
```suggestion
        .setDescription(summary.replace(/&amp;/g, '&'))
```

File: src/commands/misc/googleplay/developer.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    return `[${items.title} - ${items.developer}](${items.url})`
+  }
+
+  async handleResult ({ t, channel, author, language }, { title, screenshots, installs, released, contentRating, genre, url, priceText, price, icon, summary, appId, developer, developerId, currency }) {
+    channel.send(
+      new SwitchbladeEmbed(author)
+        .setColor(this.embedColor)
+        .setThumbnail(icon)
+        .setAuthor(title, icon, url)
+        .setDescription(summary)
```
Comment (jaxxibae) [inline src/commands/misc/googleplay/developer.js:28]:
```suggestion
        .setDescription(summary.replace(/&amp;/g, '&'))
```

**Review by jaxxibae — DISMISSED** [review]
Command doesn't error when there's no application/developer.

**Comment by Doges** [comment]
> 
> 
> Command doesn't error when there's no application/developer.
> 

searchcommand should handle that

**Review by almeidx — APPROVED** [review]

**Comment by Doges** [comment]
alo @SwitchbladeBot/chatbot-core

**Review by davipatury — APPROVED** [review]
LGTM

**Pull request merged**